packages/react/src/MegaMenu/MegaMenu.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import {
+  ForwardedRef,
+  forwardRef,
+  ForwardRefExoticComponent,
+  HTMLAttributes,
+  PropsWithChildren,
+  RefAttributes,
+} from 'react'
+
+type MegaMenuProps = {} & PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+
+interface MegaMenuComponent extends ForwardRefExoticComponent<MegaMenuProps & RefAttributes<HTMLDivElement>> {
+  ListWrapper: typeof MegaMenuListWrapper
+}
+
+export const MegaMenu = forwardRef(
+  ({ children, className, ...restProps }: MegaMenuProps, ref: ForwardedRef<HTMLDivElement>) => (
+    <nav {...restProps} ref={ref} className={clsx('amsterdam-mega-menu', className)}>
+      {children}
+    </nav>
+  ),
+) as MegaMenuComponent
+
+const MegaMenuListWrapper = forwardRef(
```

Comment:
I’m not fond of the ‘wrapper’ concept; we should find a more semantic approach in cases like these.

We can call this a ‘mega menu section’ if we move the title into it, replace the top padding with a flex gap, and set the bottom padding through `:not(:last-child)`.

Reply:
The `columns` are not going to behave nice if you put the title in there. I've added a wrapper after Aram requested me to do so and he explained you can define `columns` to the wrapper.

Reply:
Discussed with Aram, section is misleading because it is not a section tag. I think "group" is not quite right because you are not grouping lists, you're wrapping one list. So I've changed it to "ListCategory", I'm not happy about the change but I hope you can resolve it like this. Also note that the wrapper is entirely unnecessary, you can just use a `.amsterdam-mega-menu ul` selector.

Reply:
Alright, let’s have a chat about our naming approach and cooperation strategy later.

A type selector would couple CSS more tightly to the HTML. I don’t think that’s a good idea. It requires changing the CSS if the HTML gets changed, doesn’t allow e.g. an ordered list, and unintendedly targets other possible elements in the tree. We should always express our intent through classes.

Category is acceptable (or maybe Part or Region are a bit more structural?). Can you update the class name accordingly?

Reply:
Or just `MegaMenu.List`.